### PR TITLE
Optimize escaped unicode case

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/GraphQLParser/Visitors/SDLPrinter.cs [hunk 1]
```diff
@@ -1164,7 +1170,10 @@ private static async ValueTask WriteEncodedStringAsync(TContext context, ROM val
                 else if (code == '\t')
                     await context.WriteAsync("\\t").ConfigureAwait(false);
                 else
-                    await context.WriteAsync("\\u" + ((int)code).ToString("X4")).ConfigureAwait(false);
+                {
+                    await context.WriteAsync("\\u00").ConfigureAwait(false);
```
Comment (sungam3r) [inline src/GraphQLParser/Visitors/SDLPrinter.cs:1174]: We are under `code < ' '` condition and high byte is always `0`.

Comment (Shane32) [reply src/GraphQLParser/Visitors/SDLPrinter.cs:1174]: Why not add "\\u00" to each string in the array so we eliminate a call to WriteAsync?

File: src/GraphQLParser/Visitors/SDLPrinter.cs [hunk 2]
```diff
@@ -1138,6 +1138,12 @@ private static bool TryPeekParent(TContext context, [NotNullWhen(true)] out ASTN
         return true;
     }
 
+    private static readonly string[] _hex32 = new[]
+    {
+        "00", "01", "02", "03", "04", "05", "06", "07", "08", "09", "0A", "0B", "0C", "0D", "0E", "0F",
+        "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "1A", "1B", "1C", "1D", "1E", "1F"
```
Comment (Shane32) [inline src/GraphQLParser/Visitors/SDLPrinter.cs:1144]: If these are the equivalent of individual constants, shouldn't we test for all conditions?

Comment (sungam3r) [reply src/GraphQLParser/Visitors/SDLPrinter.cs]: done

File: src/GraphQLParser/Visitors/SDLPrinter.cs (hunk 1 above)
Comment (sungam3r) [reply src/GraphQLParser/Visitors/SDLPrinter.cs]: done

**Review by Shane32 — APPROVED** [review]

**Pull request merged**